lib/datadog/tracing/trace_operation.rb

```diff
@@ -72,6 +72,7 @@ def initialize(
         sampling_priority: nil,
         service: nil,
         profiling_enabled: nil,
+        apm_tracing_enabled: nil,
```

Comment:
```suggestion
        tracing_enabled: nil,
```

I suggest simplifying this to `tracing_enabled`. Even `tracing_` seems redundant here tbh, but `tracing_enabled` reads well. I think the `apm_` prefix is a bit too much, since there's no ambiguity we are operating on traces here that are part of apm.